ai_flow/metadata/workflow.py

```diff
@@ -0,0 +1,65 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from sqlalchemy import Column, String, Integer, Binary, DateTime, Boolean, BigInteger, ForeignKey, UniqueConstraint
+from sqlalchemy.orm import relationship
+
+from ai_flow.common.util.time_utils import utcnow
+from ai_flow.metadata.base import Base
+
+
+class WorkflowMeta(Base):
+    """
+    It represents the metadata of the Workflow.
+    :param id: The unique identify of the Workflow.
+    :param name: The name of the Workflow.
+    :param namespace: The namespace of the Workflow.
+    :param workflow_object: The Workflow serialized object.
+    :param content: The source code of the Workflow.
+    :param create_time: The create time of the Workflow.
+    :param update_time: The update time of the Workflow.
+    :param enable: The properties of the Workflow.
+    :param event_offset: Event processing progress corresponding to Workflow.
+    """
+    __tablename__ = 'workflow'
+
+    id = Column(Integer, autoincrement=True, primary_key=True)
+    name = Column(String(256))
+    namespace = Column(String(256),
+                       ForeignKey('namespace.name', ondelete='cascade'), nullable=False)
+    workflow_object = Column(Binary, nullable=False)
+    content = Column(String(length=1048576), nullable=False)
+    create_time = Column(DateTime)
+    update_time = Column(DateTime)
```

Comment:
Is the `update_time` duplicate with `create_time` in `workflow_snapshot`, if so, we could remove it to avoid keeping them coincident.

Reply:
remove update time